src/video2x-qt6_pt_PT.ts

```diff
@@ -0,0 +1,518 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1" language="pt_PT">
+<context>
+    <name>MainWindow</name>
+    <message>
+        <location filename="mainwindow.ui" line="14"/>
+        <source>Video2X</source>
+        <translation></translation>
+    </message>
+    <message>
+        <location filename="mainwindow.ui" line="29"/>
+        <source>Video Processing Job Queue</source>
+        <translation>Fila de Processos de Vídeo</translation>
```

Comment:
Should that be "Fila de Processamento de Videos"? Since Processing is "ING" present continuos?

Reply:
> Should that be "Fila de Processamento de Vídeo"? Since Processing is "ING" present continuos?

You're right, but the reason I put “process” is because in gui it would be a very long sentence, but Portuguese speakers can understand it anyway.

Reply:
> Should that be "Fila de Processamento de Vídeo"? Since Processing is "ING" present continuos?

You're right, but the reason I put “processo” is because in gui it would be a very long sentence, but Portuguese speakers can understand it anyway.